tests/integration/test_charm.py

```diff
@@ -23,6 +23,22 @@ async def test_active(ops_test: OpsTest, app: ops.Application):
     assert app.units[0].workload_status == status_name  # type: ignore
 
 
+@pytest.mark.asyncio
+@pytest.mark.abort_on_fail
+async def test_relation(ops_test: OpsTest, app: ops.Application, any_charm: ops.Application):
+    """Check that the charm is active once related to another charm.
+
+    Assume that the charm has already been built and is running.
+    """
```

Comment:
I think we should use the arrange,act,assert lines because of team conventions: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-structure

Reply:
Good point. Fixed